.github/workflows/release.yml

```
@@ -10,6 +10,30 @@ on:
 concurrency: ${{ github.workflow }}-${{ github.ref }}
 
 jobs:
+  cli-install-cross-platform-release-test:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
```

i just prefer using depot for anything beyond `ubuntu-latest` https://depot.dev/docs/github-actions/runner-types
```suggestion
        os: [depot-ubuntu-latest, depot-macos-latest, depot-windows-2022]
```